executor/insert_common.go

```diff
@@ -117,9 +119,30 @@ func (e *InsertValues) getColumns(tableCols []*table.Column) ([]*table.Column, e
 		return nil, errors.Trace(err)
 	}
 
+	e.initEvalBuffer()
```

Comment:
This function call can be put to `InsertExec.Open()`?

Reply:
`getColumns()` is called in the `Next()` function, Maybe we can move it to the `Open()` function:
```go
InsertExec.Open:
	getColumns()
	initEvalBuffer()

InsertExec.Next:
	e.insertRowsFromSelect(), or:
	e.insertRows()
```

Reply:
another thing is, according to the callers of the function `getColumns()`, maybe we can also improve the performance of the `replace` statement:
```
  3 executor/insert.go|132 col 17| cols, err := e.getColumns(e.Table.Cols())
  4 executor/replace.go|181 col 17| cols, err := e.getColumns(e.Table.Cols())
  5 executor/builder.go|576 col 28| columns, err := insertVal.getColumns(tableCols)
```

Reply:
finally, I move getColumns to buildExecutor just like  loadData does...and move initEvalBuffer to Open, and replace pointer to direct ref.

PTAL @zz-jason , although CI doesn't work